.github/workflows/pr_bench.yml

```diff
@@ -0,0 +1,25 @@
+name: Benchmark pull requests
+on: [pull_request]
```

Comment:
This will fire a lot, I'd reduce this to something like this:
https://github.com/lurk-lab/lurk-rs/blob/48672e8aa5baa7878584a9881126c955bf80c9ff/.github/workflows/licenses-audits.yml#L7-L8